drfx/settings.py

```diff
@@ -345,12 +345,6 @@
 # Import just to get in the translation context
 # from utils import businesslogic
 
-# Load non-default settings from settings_local.py if it exists
-try:
-    from .settings_local import *  # noqa
-except ImportError:
-    pass
-
```

Comment:
Un-delete these lines and the PR will work as intended. Perhaps leave a comment like so:

`settings_local is loaded so that  the constance config will receive defaults from settings_local.`